camel/indexing/islandora-indexing-triplestore/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>ca.islandora.camel</groupId>
+    <artifactId>islandora-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>

```

Comment:
I don't think this is necessary, but we need to add this as a sub-module [here](https://github.com/Islandora-CLAW/CLAW/blob/sprint-002/camel/pom.xml#L38-L42)